### PR TITLE
Add additional GridContainer props

### PR DESCRIPTION
**Pull request by vineethasok**
Add height, width, overflow props and also hide grid template if not passed

### CODE REVIEW

File: src/components/GridContainer/GridContainer.tsx [hunk 1]
```diff
@@ -84,36 +105,63 @@ const GridContainer = ({
...
   display: ${({ $inline }) => ($inline === true ? "inline-grid" : "grid")};
-  gap: ${({ theme, $gap }) => theme.click.gridContainer.gap[$gap]};
-  grid-auto-columns: ${({ $gridAutoColumns }) => $gridAutoColumns || "auto"};
-  grid-auto-flow: ${({ $gridAutoFlow }) => $gridAutoFlow || "auto"};
-  grid-auto-rows: ${({ $gridAutoRows }) => $gridAutoRows || "auto"};
-  grid-template-area: ${({ $gridTemplateAreas }) => $gridTemplateAreas || "auto"};
-  grid-template-columns: ${({ $gridTemplateColumns }) => $gridTemplateColumns || "auto"};
-  grid-template-rows: ${({ $gridTemplateRows }) => $gridTemplateRows || "auto"};
-  grid-template: ${({ $gridTemplate }) => $gridTemplate || "auto"};
+  ${({ $gridAutoColumns }) =>
```
Comment (fneves) [inline src/components/GridContainer/GridContainer.tsx:133]: what do oyu think of wrigint this as ```
gridAutoColumns && `grid-auto-columns: ${$gridAutoColumns}` 
```

Comment (fneves) [reply src/components/GridContainer/GridContainer.tsx:133]: and all the ternaries that end up in ""

**Review by fneves — APPROVED** [review]

File: src/components/GridContainer/GridContainer.tsx (hunk 1 above)
Comment (vineethasok) [reply src/components/GridContainer/GridContainer.tsx:133]: There is a possibility of having false added to the string

Comment (fneves) [reply src/components/GridContainer/GridContainer.tsx:133]:
```
> null && 'foo'
null
> undefined && 'foo'
undefined
```

Comment (vineethasok) [reply src/components/GridContainer/GridContainer.tsx:133]: Fixed it

**Pull request merged**